dev/buildtool/source_commands.py

```diff
@@ -138,6 +141,97 @@ def init_argparser(self, parser, defaults):
         )
 
 
+class TagBranchCommand(RepositoryCommandProcessor):
+    """Implements the tag_branch command."""
+
+    def __init__(self, factory, options):
+        """Implements CommandProcessor interface."""
+
+        self.__git = GitRunner(options)
+
+        all_names = list(SPINNAKER_RUNNABLE_REPOSITORY_NAMES)
+        all_names.extend(SPINNAKER_RUNNABLE_NON_CORE_REPOSITORY_NAMES)
+        all_names.extend(SPIN_REPOSITORY_NAMES)
```

Comment:
Not sure I understand the struggle here.  Is it that we haven't been cutting release-1.xy.x branches for spin?

Reply:
We do have `release-1.xy.z` branches for `spin` but the tag on `master` branch currently matches the Spinnaker Release Version.
For other services the tag on `master` branch is not connected (by chance or otherwise), eg: gate might be `v6.5.0`

Reply:
Is it overkill to add some other list of repos that follow the spinnaker version?  And then treat that list of repos as special in the code?  It does seem a shame to sacrifice the version matching...

Reply:
It's simple enough to add another constant but there's a catch.

`tag_branch` needs a `spinnaker_version` flag in order to tag `spin`. 
I'm not sure we want to add this so let's kick `spin` down the road a little until we tackle a revision of the `create_release_branch` command (which has `spinnaker_version` flag).

Reply:
Removed `spin` from repository targets for `tag_branch` in https://github.com/spinnaker/buildtool/pull/190/commits/04996a52be96653dc0d5d99612e0eba2f72626ea#diff-09003a65273be87edc536caabb89a3977ee30512f1c7931745ab1bf0695579e9L154